Cargo.toml

```diff
@@ -29,11 +29,12 @@ version = "0.7"
 optional = true
 
 [dev-dependencies]
-image = "0.7"
-clap = "1"
-csv = "0.14"
-rustc-serialize  = "0.3"
-lazy_static = "0.1"
+image = "0.14.0"
+clap = "2.24.0"
```

Comment:
The same for the serde dependencies. That way there's no need to update unless we want any breaking changes.